src/prefab/construct-env.ts

```diff
@@ -24,6 +24,10 @@ export default async function(pkgenv: { installations: Installation[] }) {
 
   for (const key in rv) {
     rv[key] = rv[key].replaceAll(new RegExp(`\\$${key}\\b`, 'g'), `\${${key}}`)
+    // don’t end with a trailing `:` since that is sometimes interpreted as CWD and can break things
+    // instead of `foo:${PATH}` we end up with `foo${PATH:+:PATH}` which is not POSIX but works
+    // with all realy shells to avoid the trailing `:`
+    rv[key] = rv[key].replaceAll(new RegExp(`:\\$\{${key}}`, 'g'), `\${${key}:+:$${key}}`)
```

Comment:
this is succinct, but isn't it probably safer to do:
```javascript
  .replaceAll(/::/, ':')
  .replace(/^:/, '')
  .replace(/:$/, '')
```
?

Reply:
I didn't bother with some of these as we construct this env ourselves. The runtime `env` from the package.yml could in theory cause a preceding `:` I guess.